grpc-api/src/main/java/io/deephaven/grpc_api/session/SessionState.java

```diff
@@ -597,7 +619,7 @@ public T get() {
         /**
          * @return the export id or NON_EXPORT_ID if it does not have one
```

Comment:
javadoc and method name do not match up; should we rename to `getTicket`?

Reply:
hmm, not even my fault!

I should remove the "or NON_EXPORT_ID..." right?

Reply:
I think I would state:
```
@return the ticket for this export; note if this is a non-export the received ticket will not resolve to anything and is considered an invalid ticket
```